ranger-plugin/conf/rps.properties

```diff
@@ -29,4 +29,7 @@ RPS_SHUTDOWN_PORT=8405
 RANGER_HAWQ_INSTANCE=hawq
 
 # version of the software
-RPS_VERSION=${project.version}
\ No newline at end of file
+RPS_VERSION=${project.version}
+
+# home path of Tomcat/6.0.x for Ranger Plugin Service
+CATALINA_HOME=
```

Comment:
We could document this for community users and other users that install from RPM. 
But it's okay to make a change. 
I will move the definition of CATALINA_HOME to rps.properties and leave a placeholder for community users to add their own path.